package.json

```
@@ -12,15 +12,20 @@
     "code_format": "run-s code_format:prettier code_format:svgo",
     "code_format:prettier": "prettier --write --list-different .",
     "code_format:svgo": "svgo -q -f icons/",
-    "clean": "shx rm -rf dist",
+    "clean": "run-s clean-download clean-build",
+    "clean-download": "shx rm -rf download",
+    "clean-build": "shx rm -rf dist build",
+    "fonts:install_glyph_builder": "cargo install build_pbf_glyphs@1.3.0",
+    "fonts:build": "node scripts/fonts",
+    "fonts": "run-s fonts:install_glyph_builder fonts:build",
     "presprites": "shx rm -rf dist/sprites",
     "sprites": "node scripts/sprites.js",
     "status_map": "node scripts/status_map.js",
     "taginfo": "node scripts/taginfo.js",
-    "start": "run-s clean sprites serve",
+    "start": "run-s clean-build sprites fonts serve",
     "serve": "node scripts/serve.js",
     "build:code": "node scripts/build.js",
-    "build": "run-s clean sprites build:code taginfo status_map",
+    "build": "run-s clean sprites fonts build:code taginfo status_map",
```

As with the `start` script above, building the project shouldn’t toss the downloaded fonts. This change will eliminate the build overhead I was complaining about earlier:

```suggestion
    "build": "run-s clean-build sprites fonts build:code taginfo status_map",
```